v1/src/test/java/com/google/cloud/teleport/spanner/AvroRecordConverterTest.java

```diff
@@ -258,6 +258,103 @@ public void testParseFloat() {
     assertThrows(IllegalArgumentException.class, () -> avroRecordConverter.apply(avroRecord1));
   }
 
+  @Test
+  public void float32Array() {
```

Comment:
Nit: Please name the method according to what it is testing

Reply:
Done